app-admin/webmin/webmin-1.791.ebuild

```diff
@@ -285,7 +285,7 @@ pkg_config(){
 	export perl="$( which perl )"
 	export os_type='gentoo-linux'
 	export os_version='*'
-	export real_os_type='Gentoo Linux'
+	export real_os_type="${DISTRO} Linux"
```

Comment:
I'm not sure the best way to proceed with this; waiting for reply to my comment first.